KeenClientTests/KeenClientTests.m

```diff
@@ -1964,7 +1988,7 @@ - (NSString *)cacheDirectory {
 }
 
 - (NSString *)keenDirectory {
-    return [[[self cacheDirectory] stringByAppendingPathComponent:@"keen"] stringByAppendingPathComponent:@"id"];
+    return [[[self cacheDirectory] stringByAppendingPathComponent:@"keen"] stringByAppendingPathComponent:kDefaultProjectID];
 }
```

Comment:
This file is >2000 lines. Similar to what we plan to do with the Java SDK tests, we should look into splitting this apart so it's not quite so unwieldy. Maybe create an Issue? Or just do it at some point... :)